### PR TITLE
#354,346 expand assistant privileges and csv download improvements

### LINKED ISSUE #354: Add Non-UM user assistant should have same role adding privileges as Teacher
`Assistant` should be able to add `Teacher, Student, Designer, TA, Observer, Librarian, Assistant`, Current setup only allows student

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #354 #346

### CODE REVIEW

**Comment by pushyamig** [comment]
This may affect the Add-Non UM user integration with BE

File: ccm_web/client/src/components/SingleSectionEnrollmentWorkflow.tsx [hunk 1]
```diff
@@ -213,11 +213,7 @@ export default function SingleSectionEnrollmentWorkflow (props: SingleSectionEnr
 
     const fileData =
     `${REQUIRED_ENROLLMENT_HEADERS.join(',')}
-    student,studentone
-    teacher,teacherone
-    ta,taone
-    observer,observerone
-    designer,designerone`
+     studentone, student`
```
Comment (ssciolla) [inline ccm_web/client/src/components/SingleSectionEnrollmentWorkflow.tsx:216]: Thanks for finding (@melindakraft) and fixing (@pushyamig) my mistake!

See note in other comment about leading whitespace.

File: ccm_web/client/src/components/MultipleSectionEnrollmentWorkflow.tsx [hunk 2]
```diff
@@ -185,15 +185,9 @@ export default function MultipleSectionEnrollmentWorkflow (props: MultipleSectio
...
-      'teacherone,teacher,1001\n' +
-      'teacherone,teacher,1002\n' +
-      'studentone,student,1001\n' +
-      'studenttwo,student,1002\n' +
-      'taone,ta,1001\n' +
-      'tatwo,ta,1002\n'
-    )
+    const fileData =
+      `${REQUIRED_ENROLLMENT_WITH_SECTION_ID_HEADERS.join(',')}
+       teacherone,teacher,1001`
```
Comment (ssciolla) [inline ccm_web/client/src/components/MultipleSectionEnrollmentWorkflow.tsx:190]: This is still a valid CSV, but there's unnecessary whitespace in front of the text in the first cell in the non-header row. This is true of the other one you changed too. You could either unindent 190 entirely, or do something like I did in the removed lines where you concatenate with `\n`. I prefer the latter since it preserves code indentation, but up to you.

**Review by ssciolla — APPROVED** [review]
Thanks! It would be nice to address the leading whitespace issue (just makes the sample file neater), but no need to re-request a review.

File: ccm_web/client/src/components/SingleSectionEnrollmentWorkflow.tsx (hunk 1 above)
Comment (pushyamig) [reply ccm_web/client/src/components/SingleSectionEnrollmentWorkflow.tsx]: Done

File: ccm_web/client/src/components/MultipleSectionEnrollmentWorkflow.tsx (hunk 2 above)
Comment (pushyamig) [reply ccm_web/client/src/components/MultipleSectionEnrollmentWorkflow.tsx]: Done
